fog/ledger/enclave/api/src/lib.rs

```diff
@@ -98,6 +101,62 @@ pub trait LedgerEnclave: ReportableEnclave {
 
     /// Add a key image data to the oram Using thrm -rf targete key image
     fn add_key_image_data(&self, records: Vec<KeyImageData>) -> Result<()>;
+
+    // LEDGER ROUTER / STORE SYSTEM
+
+    /// Begin a connection to a Fog Ledger Store. The enclave calling this
+    /// method, most likely a router, will act as a client to the Fog Ledger
+    /// Store.
+    fn ledger_store_init(&self, ledger_store_id: ResponderId) -> Result<NonceAuthRequest>;
+
+    /// As a "Store" in a Router/Store system,
+    /// accept a connection from a Router.
+    fn frontend_accept(
+        &self,
+        auth_request: NonceAuthRequest,
+    ) -> Result<(NonceAuthResponse, NonceSession)>;
+
+    /// Complete the connection to a Fog Ledger Store that has accepted our
+    /// NonceAuthRequest. This is meant to be called after the enclave has
+    /// initialized and discovers a new Fog Ledger Store.
+    fn ledger_store_connect(
+        &self,
+        ledger_store_id: ResponderId,
+        ledger_store_auth_response: NonceAuthResponse,
+    ) -> Result<()>;
+
+    /// Extract context data to be handed back to untrusted so that it could
```

Comment:
nit: What context data is this referring to? The comment seems a bit generic, and, as someone who isn't intimately familiar with the check_key_image_store method, it'd be helpful to know what the method is doing rather than how it's doing it.

Reply:
Addressed in [278d8c1](https://github.com/mobilecoinfoundation/mobilecoin/pull/2896/commits/278d8c1f98696a964ebd09ae87c4b2d5e316cfd0)